### PR TITLE
edited roxygen2

### PR DESCRIPTION
**Pull request by tomoya-sasaki**

### CODE REVIEW

**Comment by Shusei-E** [comment]
The base branch is `v0.2.0`. Could you double-check the conflicts?

**Comment by Shusei-E** [comment]
Also, tests failed. Is this branched out from `v0.2.0`?

**Comment by tomoya-sasaki** [comment]
I have merged from `package_dev` and commit again to `package_dev`. I am not sure why it did not pass the test...

**Comment by tomoya-sasaki** [comment]
I will recheck and recommit this

**Comment by Shusei-E** [comment]
> I have merged from package_dev and commit again to package_dev. I am not sure why it did not pass the test

The latest version is in `v0.2.0` branch.

**Comment by tomoya-sasaki** [comment]
Yeah, I now realized that

**Comment by Shusei-E** [comment]
I updated documentation as well.
https://github.com/keyATM/keyATM/projects/1

File: R/keyATM.R [hunk 1]
```diff
@@ -65,20 +65,23 @@
 #'   keyATM_docs <- keyATM_read(bills_dfm)
 #'
 #'   # keyATM Base
-#'   out <- keyATM(keyATM_docs, model = "base", no_keyword_topics = 5,
-#'                 keywords = bills_keywords)
+#'   out <- keyATM(docs = keyATM_docs,
+#'                 model = "base", no_keyword_topics = 5,
```
Comment (Shusei-E) [inline R/keyATM.R:69]: Where is the keyword argument?

**Pull request merged**